ch_tools/common/logging.py

```diff
@@ -0,0 +1,182 @@
+"""
+Logging module.
+"""
+
+import inspect
+import logging
+import sys
+from typing import Any, Optional
+
+from loguru import logger
+
+# Used to store this.module variable
+this = sys.modules[__name__]
```

Comment:
Do we really need to put logger name in `sys.modules`? I think we can replace it with  dictionary to keep name and option parameters. besides, we'll get rid of replacing parameters in the format string: https://github.com/kirillgarbar/ch-tools/blob/4df613ecf6a1c8d7cf683205945cbacfa80ee7d6/ch_tools/common/logging.py#L80